### PR TITLE
[11.0][ADD] stock_request_purchase_analytic

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
This module integrates #519 with stock_request_purchase module. Basically, when a Purchase Order is created from a stock request or a stock request order, the analytic account is passed to the purchase order line.

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
This is ready to review. It seems there is some load issue in runbot, I'll restart the build.

File: stock_request_purchase_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Jordi Ballester <jordi.ballester@eficent.com>.
+* Enric Tobella <etobella@creublanca.es>
```
Comment (LoisRForgeFlow) [inline stock_request_purchase_analytic/README.rst:44]: what about you?

File: stock_request_purchase_analytic/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
```
Comment (LoisRForgeFlow) [inline stock_request_purchase_analytic/__manifest__.py]: 2019? :thinking:

**Review by LoisRForgeFlow — COMMENTED** [review]
Some very minor comments, but it looks good.

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@lreficent

**Comment by etobella** [comment]
LGTM :+1:

**Review by etobella — APPROVED** [review]

**Pull request merged**